lib/util/supabase.dart

```diff
@@ -26,4 +36,68 @@ class SupabaseManager {
       _currentProfile = null;
     }
   }
+
+
```

Comment:
I'm sorry, WHAT?

- what if I want to use three "eq" operators? Or any operator not used here (there are a lot of filter operators, for example)? Do I need to write a new method for any combination of operators I want to use in any place? We can't do that.
- the arguments here could easily be named hash arguments like we use everywhere. Otherwise you have no clue what to put where when you call the method.
- all of these methods should be static, why are they instance methods?
- "getOf" -> "getFrom" and maybe "selectFrom" is better anyways

I understand you needed to write these methods for mocking, but I'm sure there is a better way.